cypress/integration/helpAndResources.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Looking at the Help and Resources gutter', () => {
+  it('Opens the app', () => {
+    cy.visit(Cypress.env('EDITOR_URL') || 'http://localhost:8000/')
+    cy.contains('The underdrawing for the new world of linked data in libraries')
+  })
+
+  it('Opens help and resources', () => {
+    cy.contains('a', 'Help and Resources').click()
+
+    cy.contains('External Identifier Sources and Vocabularies')
+  })
+
+  it('Closes help and resources', () => {
+    cy.contains('a', 'Help and Resources').click()
+
+    cy.contains('External Identifier Sources and Vocabularies')
+  })
+
+  it('Logs in', () => {
+    // Requires that COGNITO_TEST_USER_NAME and COGNITO_TEST_USER_PASS be available as env variables.
+    // See https://docs.cypress.io/guides/guides/environment-variables.html
+    cy.get('#username')
+      .type(Cypress.env('COGNITO_TEST_USER_NAME'))
+      .should('have.value', Cypress.env('COGNITO_TEST_USER_NAME'))
+    cy.get('#password')
+      .type(Cypress.env('COGNITO_TEST_USER_PASS'))
+      .should('have.value', Cypress.env('COGNITO_TEST_USER_PASS'))
+    cy.contains('Login').click()
+    cy.contains('current cognito user: sinopia-devs_client-tester')
+  })
+
+  it('Opens Linked Data Editor', () => {
```

Comment:
I noticed that the issue specified that the Help and Resources menu could be opened from each of the Sinopia tabs, but this test only tests the Resource Templates tabs. @justinlittman, is this sufficient?

Reply:
Each of tabs need to be tested separately. (The bug that initiated this was only on some tabs.)

Reply:
Covering every permutation of features (thinking broader than the issue at hand) in cypress is going to create a very slow test suite.  Can we just cover the cases that caused the trouble?

Reply:
Philosophically I agree with you. However, in this case we want to make sure that a dev doesn't break the menu for any tab (as there are 2 places in code each tab can be broken, and it is not immediately obvious). So this isn't just a case of "test everything" but rather test a specific vulnerability due to the structure of the code that actually occurred (I broke some of the tabs, not all.)